Makefile

```
@@ -4,7 +4,7 @@ test:
 	python -m pytest -n auto --dist=loadfile -s -v ./tests/
 
 quality:
-	black --check --line-length 119 --target-version py38 tests trl
+	black --check --line-length 119 --target-version py38 tests trl examples
 	isort --check-only tests trl
 	flake8  tests trl
 
```

Let's apply all tests to the same files:

```suggestion
	black --check --line-length 119 --target-version py38 tests trl examples
	isort --check-only tests trl examples
	flake8  tests trl examples

```

---

Let's do the same in `style` and remove `setup.py`

---

Proposed something in [4ad0581](https://github.com/lvwerra/trl/pull/197/commits/4ad05816af684111d5a867e43db086fffb2b37f8)

---

much cleaner indeed :)